target/i386/hax-all.c

```diff
@@ -506,6 +508,57 @@ void hax_raise_event(CPUState *cpu)
     vcpu->tunnel->user_event_pending = 1;
 }
 
+static inline int target_memory_rw_debug(CPUState *cpu, target_ulong addr,
+                                         uint8_t *buf, int len, bool is_write)
+{
+    CPUClass *cc = CPU_GET_CLASS(cpu);
+
+    if (cc->memory_rw_debug) {
+        return cc->memory_rw_debug(cpu, addr, buf, len, is_write);
+    }
+    return cpu_memory_rw_debug(cpu, addr, buf, len, is_write);
+}
+
+static void hax_update_orbital_cpu_procs(CPUArchState *env, CPUState *cpu) {
+    struct hax_msr_data md;
+    struct vmx_msr *msrs = md.entries;
+    int ret, n;
+    const uint64_t PROC_NAME_LEN = 20;
+
+    n = 0;
+    msrs[n++].entry = MSR_GSBASE;
+    md.nr_msr = n;
+    ret = hax_sync_msr(env, &md, 0);
+
+    if (ret >= 0) {
```

Comment:
Can you please change this for:

```suggestion
     if (ret < 0) {
        fprintf(stderr, "err syncing msrs for vcpu %x\n", cpu->cpu_index);
        return;
    }
```

This saves unnecessary indentation.